### PR TITLE
Change docs destination repository [doc only]

### PR DESCRIPTION
**Pull request by porunov**
Signed-off-by: Oleksandr Porunov <alexandr.porunov@gmail.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?
- [x] If this PR is a documentation-only change, have you added a `[doc only]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -189,4 +190,4 @@ jobs:
 # Syntax and more info: https://docs.travis-ci.com/user/notifications
 notifications:
   email:
-    - janusgraph-ci@googlegroups.com
\ No newline at end of file
+    - janusgraph-ci@googlegroups.com
```
Comment (farodin91) [inline .travis.yml]: Why do removed the last line?

Comment (porunov) [reply .travis.yml]: I didn't remove anything. IntelijIDEA adds new line symbol by default on the end of each file. I.e. the line was not removed. IntelijIDEA just added a hidden character. If necessary I can remove it but I think this is OK

Comment (farodin91) [reply .travis.yml]: Okay

Comment (mbrukman) [reply .travis.yml]: I'm guessing what happened here is that this file was modified previously by someone with a different setting for newline at EOF, and we'll have these spurious diffs every time someone with a different configuration setting updates a file.

If you'd like consistent editor settings across everyone's environments, please update `https://github.com/JanusGraph/janusgraph/blob/master/.editorconfig` and set up your editor accordingly, e.g., there's a [plugin for IntellijIDEA](https://github.com/JetBrains/intellij-community/tree/master/plugins/editorconfig) which supports this config file, and it's supported for a lot of other editors as well: https://editorconfig.org/#download

Comment (porunov) [reply .travis.yml]: @mbrukman Thank you. I configured my IDEA to not change those files. I think this won't happen again.

**Pull request merged**